src/templates/partials/top.ejs

```diff
@@ -6,7 +6,7 @@
   <meta charset="utf-8">
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1">
-  <title>Welcome to the Pylons Project Organization</title>
+  <title><% if (context.index_page) { %>Welcome to the Pylons Project Organization<% } else { %><%= context.title %> | The Pylons Project<% } %>
```

Comment:
* "Welcome to the Pylons Project", without "Organization"
* " | Pylons Project", without "The "

We've never had "Organization" in our name, and "The " does not add to SEO.